docs/pages/versions/unversioned/guides/configuring-statusbar.md

```diff
@@ -13,35 +13,57 @@ The configuration for Android status bar lives under the `androidStatusBar` key
 This option can be used to specify whether the status bar content (icons and text in the status bar) is light, or dark. Usually a status bar with a light background has dark content, and a status bar with a dark background has light content.
 
 The valid values are:
-
 - `light-content` - The status bar content is light colored (usually white). This is the default value.
 - `dark-content` - The status bar content is dark colored (usually dark grey). This is only available on Android 6.0 onwards. It will fallback to `light-content` in older versions.
 
+> Note: Be aware that choosing `light-content` and having `SplashScreen` image that is very light (close to `white` color) and/or having `backgroundColor` of your app set to `white` (or close to `white` color) may result in status bar's icons blending in and wouldn't be visible.
+
 ### `backgroundColor`
 
 This option can be used to set a background color for the status bar.
 
-Keep in mind that the Android status bar is translucent by default in Expo apps. But, when you specify an opaque background color for the status bar, it'll lose it's translucency.
+Keep in mind that the Android status bar can be `translucent` (default behavior for Expo apps).
+But, when you specify an opaque background color for the status bar, it'll lose it's translucency.
+
+The valid value is a 6-character long hexadecimal solid color string of shape `#RRGGBB` (e.g. `#C2185B`) or 8-character long hexadecimal color string with transparency of shape `#RRGGBBAA` (e.g. `#23C1B255`).
+
+### `translucent`
+
+Value type - `boolean`.
+Specifies whether status bar should be translucent.
+Translucent status bar is visible on the screen, but it takes no space and your application can draw beneath it (similar to element with styles `{ position: "absolute", top: 0 }` that is rendered above the app content at the top of the screen).
+Non-translucent status bar behaves as a block element and limits space available on your device's screen.
+Defaults to `true`.
+
+> Note: Translucent status bar makes sense when it's `backgroundColor` is transparent (at least to some extend).
```

Comment:
```suggestion
> Note: A translucent status bar makes sense when the `backgroundColor` is transparent.
```

Reply:
I'm not sure I understand what this means

Reply:
@cruzach I've added a little more explanation what's on my mind here.